### DISCUSSION TITLE
Version 4.3.4

### DISCUSSION THREAD

**Discussion by dylanpiera** [pokemon-tabletop-reunited/ptr1e#796] [General]
@righthandofvecna

Planning to take a look at your work on Friday, merge it all in assuming no issues arise, and release the patch.

FYI are you in our [discord](https://discord.gg/ptrfvtt)?

**Comment by righthandofvecna** [comment]
Sweet, thanks :D I think #790 and #792 will have merge conflicts, but it's a pretty simple fix.

I am in the discord, just lurking a bit. `@dylan.is.super.ok`
